src/js/view/controls/settings-menu.js

```diff
@@ -83,6 +86,7 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
     const onQualitiesChanged = (changedModel, levels) => {
         if (!levels || levels.length <= 1) {
             removeQualitiesSubmenu(settingsMenu);
+            showSettingsMenuIcon(settingsMenu, controlbar);
```

Comment:
instead of having the same function call for both scenarios, i would wrap the bottom condition in an else statement and call the function once after the if-else block
```
    const onQualitiesChanged = (changedModel, levels) => {
        if (!levels || levels.length <= 1) {
            removeQualitiesSubmenu(settingsMenu);
        } else {
            const { hd, auto } = model.get('localization');
            addQualitiesSubmenu(
                settingsMenu,
                levels,
                (index) => api.setCurrentQuality(index),
                model.get('currentLevel'),
                hd,
                auto
            );
        }
        showSettingsMenuIcon(settingsMenu, controlbar);
    };
```